addon/-private/column-tree.js

```diff
@@ -104,29 +107,47 @@ class ColumnTreeNode {
       // watchers
       this._notifyMaxChildDepth = () => notifyPropertyChange(parent, 'maxChildDepth');
       this._notifyLeaves = () => notifyPropertyChange(parent, 'leaves');
+      this._notifyCollection = () => notifyPropertyChange(parent, '[]');
+
       addObserver(this, 'maxChildDepth', this._notifyMaxChildDepth);
       addObserver(this, 'leaves.[]', this._notifyLeaves);
     }
   }
 
   destroy() {
-    removeObserver(this, 'maxChildDepth', this._notifyMaxChildDepth);
-    removeObserver(this, 'maxChildDepth', this._notifyLeaves);
```

Comment:
Did this introduce a different memory leak? I think the second line was wrong originally (probably meant it to be `removeObserver(this, 'leaves.[]', this._notifyLeaves);` but how are these observers getting cleaned up without them?